bench.py

```diff
@@ -175,7 +175,11 @@ def parse_args(parser, size=None, loop_types=(),
                         help='Dataset name')
     parser.add_argument('--no-intel-optimized', default=False, action='store_true',
                         help='Use no intel optimized version. '
-                             'Now avalible for scikit-learn benchmarks'),
+                             'Now avalible for scikit-learn benchmarks')
+    parser.add_argument('--device', default=None, type=str,
```

Comment:
```suggestion
    parser.add_argument('--device', default="host", type=str,
```

Reply:
My understanding is that `None` is used to run without context. Other values specify device type for a context

Reply:
ohh, then I think the host is not needed at all